core/src/main/scala/com/velocidi/apso/json/Implicits.scala

```diff
@@ -220,4 +220,21 @@ object Implicits {
         createJson(path.split(separatorRegex).toList, value).deepMerge(fromCirceFullPaths(rem, separatorRegex))
     }
   }
+
+  final implicit class ApsoJsonEncoder[A](val encoder: Encoder[A]) extends AnyVal {
+    /**
+     * Returns an encoder that removes the json fields that are null.
```

Comment:
Can you mention here that this won't recursively traverse all objects and only apply to the root?